sample/app/http-env-echo-no-loadbalancer.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Can you set this up in https://github.com/pebble/spacel-provision/blob/master/src/test_integ/test_deploy.py too? Decent-sized feature, we probably want automated test coverage.
It may be too annoying (i.e. custom-code just to find the public IPs) to verify before #50 though...

(maybe test_ssh.py too to verify public SSH path when there's no bastion?)